doc/releases/release-notes-4.0.rst

```diff
@@ -721,16 +721,21 @@ Drivers and Sensors
   * Introduced API for partial frames transfer with the video buffer field ``line_offset``
   * Introduced API for :ref:`multi-heap<memory_management_shared_multi_heap>` video buffer allocation with
     :kconfig:option:`CONFIG_VIDEO_BUFFER_USE_SHARED_MULTI_HEAP`
-  * Introduced bindings for common video link properties in ``video-interfaces.yaml``
+  * Introduced bindings for common video link properties in ``video-interfaces.yaml``. Migration to the
+    new bindings is tracked in (:github:`80514`)
   * Introduced missing :kconfig:option:`CONFIG_VIDEO_LOG_LEVEL`
   * Added a sample for capturing video and displaying it with LVGL
     (:zephyr:code-sample:`video-capture-to-lvgl`)
+  * Added an automatic test to check colorbar pattern correctness
   * Added support for GalaxyCore GC2145 image sensor (:dtcompatible:`gc,gc2145`)
   * Added support for ESP32-S3 LCD-CAM interface (:dtcompatible:`espressif,esp32-lcd-cam`)
   * Added support for NXP MCUX SMARTDMA interface (:dtcompatible:`nxp,smartdma`)
   * Added support for more OmniVision OV2640 controls (:dtcompatible:`ovti,ov2640`)
   * Added support for more OmniVision OV5640 controls (:dtcompatible:`ovti,ov5640`)
   * STM32: Implemented :c:func:`video_get_ctrl` and :c:func:`video_set_ctrl` APIs.
+  * Fixed a chicken-egg issue in init orders of the camera pipeline on NXP RT10xx platforms
+    (:github:`80304`)
+  * Fixed a floating point logging issue in the video capture sample (:github:`80653`)
```

Comment:
I would drop these two as they are very specific and probably not worth mentioning in the global release notes imo

Reply:
Okay, the last one is minor, I will drop it.

But the chicken-egg issue in init order is an issue that make camera not work on on NXP 10xx platforms upstream for a long time. It seems worth mentionning to customers that camera is now working in upstream on those platforms ?

Also, this can happen to any camera pipeline where the receiver (e.g. CSI) need to be initialized before the sensor to provide clock to the sensor but if we change the receiver init priority (AFAIK at least Mediatek 8183 SoC has the same dependency, which is not yet in Zephyr though), it will not compile because the CSI node refers to the camera sensor node in the DT so it has to be initialized AFTER the sensor.

This is not a bug but rather a limitation in Zephyr DT which does not support circular dependency (endpoint / remote-endpoint). This is resolved with "remote-endpoint-label" in the new video-interfaces bindings.

Sorry for the long comment but I would like to know if it's worth mentionning in the upcomming Zephyr techtalk too, if you say no, I will drop it then.

Reply:
Would the first be better worded as `Removed an init order circular dependency for the camera pipeline`

Reply:
@dkalowsk : yes, will do. Thanks.

Reply:
Changed the wording, waiting for @kartben 's advice if it's worth to add the "chicken-egg" issue in init order